client/src/entrypoints/admin/core.js

```diff
@@ -32,6 +35,21 @@ window.wagtail = wagtail;
 
 window.escapeHtml = escapeHtml;
 
+/**
+ * Support legacy global URLify which can be called with `allowUnicode` as a third param.
+ * Was not documented and only used in modeladmin prepopulate JS.
+ *
+ * @deprecated RemovedInWagtail70
+ * @see https://github.com/django/django/blob/main/django/contrib/admin/static/admin/js/urlify.js#L156
+ *
+ * @param {string} str
+ * @param {number} numChars
+ * @param {boolean} allowUnicode
+ * @returns {string}
+ */
+window.URLify = (str, numChars = 255, allowUnicode = false) =>
+  urlify(str, { numChars, allowUnicode }) || slugify(str, { allowUnicode });
```

Comment:
Just want to note that the fallback to `slugify` wasn't in the original behaviour. I don't think it hurts to add it here, though, so I'm happy to leave it as-is.

Reply:
ahh good call, this is for the deprecation function so probably not a huge deal.

Thanks for the approval, I will remove this when merging in.